packages/core/src/tasks/index.ts

```diff
@@ -358,9 +358,15 @@ export const proposeAbstractTask = async (
   const newCanonicalConfig: CanonicalConfig = {
     manager: prevConfig.manager,
     options: parsedConfig.options,
```

Comment:
I think we may need to combine the `options.chainIds` too. In the example you showed, it seems like the first chain ID (arbitrum goerli) would be removed from this field

Reply:
Are you sure? The `ConfigOptions` type does not appear to include the `chainIds` field: 
https://github.com/sphinx-labs/sphinx/blob/804309f8cc33d2a9a3fcabff081870daf63bbdb7/packages/core/src/config/types.ts#L120

Only the `ParsedConfigOptions` type includes that field, but it is not used here.